src/jmcomic/jm_async_client.py

```
@@ -708,20 +734,22 @@ async def setup(self):
+            if self._has_setup:
+                return
+
+            if not cls._has_setup_domain:
                 await self.auto_update_domain()
-                if JmModuleConfig.FLAG_API_CLIENT_REQUIRE_COOKIES:
-                    await self.ensure_have_cookies()
-                cls._has_setup_domain_and_cookies = True
+                cls._has_setup_domain = True
             else:
-                # 即使已经初始化过域名和 cookie，也需要将已保存的全局 DOMAIN 和 COOKIES 赋值到当前 client
+                # 即使已经初始化过域名，也需要将已保存的全局 DOMAIN 赋值到当前 client
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)